docs/manual/output.md

```diff
@@ -392,19 +392,6 @@ output ntuples as a
 To use this feature, simply specify an output file with a `.lh5` extension, and
 _remage_ will perform the file conversion automatically.
 
-:::{note}
```

Comment:
you removed the opening tag of a note that is closed much later after the next paragraph... this should be restored